CHANGELOG.md

```diff
@@ -14,7 +14,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased 2.x](https://github.com/opensearch-project/k-NN/compare/2.13...2.x)
 ### Features
-* Support radial search in k-NN plugin [#1617](https://github.com/opensearch-project/k-NN/pull/1617)
+* Support radial search in k-NN plugin [#814](https://github.com/opensearch-project/k-NN/issues/814)
```

Comment:
why we're changing PR ref?

Reply:
I intended to use the feature issue as entrypoint here, will update to have a new line for this PR.

Reply:
Updated